public/app/features/alerting/unified/components/silences/SilencesEditor.tsx

```diff
@@ -89,14 +87,16 @@ const getDefaultFormValues = (queryParams: UrlQueryMap, silence?: Silence): Sile
       matcherName: '',
       matcherValue: '',
       timeZone: DefaultTimeZone,
-      ...defaultsFromQuery(queryParams),
+      ...defaultsFromQuery(searchParams),
     };
   }
 };
 
 export const SilencesEditor: FC<Props> = ({ silence, alertManagerSourceName }) => {
-  const [queryParams] = useQueryParams();
-  const defaultValues = useMemo(() => getDefaultFormValues(queryParams, silence), [silence, queryParams]);
+  const [urlSearchParams] = useURLSearchParams();
+  console.log(urlSearchParams);
```

Comment:
ops :)

Reply:
Fixed. Shouldn't Eslint catch it on pre-commit or pre-push actions?

Reply:
It looks like those are allowed by default https://github.com/grafana/eslint-config-grafana/blob/master/index.js#L35

A quick code grep shows quite a few existing uses of `console.<fn>` spread throughout the `public/` folder.